### PR TITLE
Adjust e2e browser based tests

### PR DESCRIPTION
**Pull request by garciafdezpatricia**
# New feature description
The Dev-2-3 environment introduces a restriction on the clients that can issue an Access Grant, breaking e2e tests whose target environment is Dev-2-3 since they're not acting as an allowed client. 
The adjustment replaces the use of access grants with access requests, which don't have any restriction and don't need AMC to be involved in the process: instead of approving and revoking an access grant, an access request is issued and denied.
The test-app UI has also been modified according to the changes.

# Checklist

- [ ] All acceptance criteria are met.
- [ ] Relevant documentation, if any, has been written/updated.
- [ ] The changelog has been updated, if applicable.
- [ ] New functions/types have been exported in `index.ts`, if applicable.
- [ ] New modules (i.e. new `.ts` files) are listed in the `exports` field in `package.json`, if applicable.
- [ ] New modules (i.e. new `.ts` files) are listed in the `typedocOptions.entryPoints` field in `tsconfig.json`, if applicable.
- [x] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).

### CODE REVIEW

File: e2e/browser/test-app/components/accessGrants/index.tsx
Comment (NSeydoux) [inline e2e/browser/test-app/components/accessGrants/index.tsx:51]: Let's not have an ambiguous type like this, and either keep it a plain string or an `AccessRequest` object.

File: e2e/browser/test-app/components/accessGrants/index.tsx
Comment (NSeydoux) [inline e2e/browser/test-app/components/accessGrants/index.tsx:102]: This test could be added to the clause of line 97. It's usually more readable to have an early return (and so to test for negative conditions) rather than testing for positive conditions and having the "main" code inside the conditional branch.

File: e2e/browser/test-app/components/accessGrants/index.tsx
Comment (NSeydoux) [inline e2e/browser/test-app/components/accessGrants/index.tsx:105]: `requestor` is a deprecated parameter and should be removed.

File: e2e/browser/test-app/components/accessGrants/index.tsx
Comment (NSeydoux) [inline e2e/browser/test-app/components/accessGrants/index.tsx:113]: If this isn't strictly required, let's not include it at all and use the defaults.

File: e2e/browser/test-app/components/accessGrants/index.tsx
Comment (NSeydoux) [inline e2e/browser/test-app/components/accessGrants/index.tsx:131]: Denying the Access grant is a privileged operation that issues an Access Grant, it has to be performed by an authorized client. This should cancel the Access Request instead.

**Review by NSeydoux — CHANGES_REQUESTED** [review]
I know cancelling the Access Request is currently blocked by a bug. Let's pause this PR for the time being, fix the bug, and then rework this once the fix has been merged.

**Comment by edwardsph** [comment]
This has been incorporated into https://github.com/inrupt/solid-client-access-grants-js/pull/999

**Pull request closed without merging**
